tests/test_kafka_scanner.py

```diff
@@ -202,7 +226,7 @@ def test_kafka_scan_dedupe_many(self, kafka_consumer_mock):
         self.assertEqual(scanner.scanned_count, 2000)
         self.assertEqual(scanner.dupes_count, 1000)
 
-    def test_kafka_scan_lower_offsets(self, kafka_consumer_mock, batchsize=10000):
+    def _test_kafka_scan_lower_offsets(self, kafka_consumer_mock, batchsize=10000):
```

Comment:
which is the point of this kind of change and why to introduce it in a PR for adding Py3 support?